app/subscriber/src/features/my-reports/edit/styled/ReportEditForm.tsx

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 export const ReportEditForm = styled.div`
   display: flex;
   flex-direction: column;
-  overflow-x: auto;
+  overflow-x: clip;
```

Comment:
@kkwangsir This line broke the stick footers that provide the action buttons for each form.  Switching from `auto` to `clip` appears to work.  Can you provide the reason for adding `overflow-x: auto` here?